### PR TITLE
Create ClearCapture method in ClientGgp

### PR DESCRIPTION
**Pull request by kisenshi**
Create new method that allows to clear the capture data and
the variables related to it. It is called before a capture is started

### CODE REVIEW

File: OrbitClientGgp/ClientGgp.cpp [hunk 1]
```diff
@@ -76,10 +76,12 @@ bool ClientGgp::RequestStartCapture(ThreadPool* thread_pool) {
     return false;
   }
 
+  // Clean capture data and variables to start from a clean state
```
Comment (ronaldfw) [inline OrbitClientGgp/ClientGgp.cpp:79]: I would remove this comment. It doesn't really help understanding this better. ClearCapture() is a pretty clear name.

File: OrbitClientGgp/ClientGgp.cpp [hunk 2]
```diff
@@ -76,10 +76,12 @@ bool ClientGgp::RequestStartCapture(ThreadPool* thread_pool) {
...
+  ClearCapture();
+
   // Start capture
```
Comment (ronaldfw) [inline OrbitClientGgp/ClientGgp.cpp:82]: Not added in this PR, but this comment should also be removed.

**Review by ronaldfw — APPROVED** [review]
LGTM, just minor comments on comments.

File: OrbitClientGgp/ClientGgp.cpp (hunk 1 above)
Comment (kisenshi) [reply OrbitClientGgp/ClientGgp.cpp]: You are right, I have removed it, thanks!

File: OrbitClientGgp/ClientGgp.cpp (hunk 2 above)
Comment (kisenshi) [reply OrbitClientGgp/ClientGgp.cpp]: Done!

**Comment by kisenshi** [comment]
I have removed the unnecessary comments. Waiting for all the checks to complete to merge. Thanks!

**Comment by kisenshi** [comment]
Merging now.

**Pull request merged**